back/src/Model/GameRoom.ts

```diff
@@ -249,32 +246,64 @@ export class GameRoom {
                 }
             }
         } else {
-            // If the user is part of a group:
-            //  should he leave the group?
-            let noOneOutOfBounds = true;
-            group.getUsers().forEach((foreignUser: User) => {
-                if (foreignUser.group === undefined) {
-                    return;
+            let headMembers = group.getGroupHeads();
+            let hasKickOutSomeone = false;
+
+            const previewNewGroupPosition = previewGroupPosition(headMembers);
```

Comment:
previewGroupPosition is **always** called with the headMembers parameters.

So rather that making a `previewGroupPosition` function, could we make a `previewGroupPosition` method?

Like:

```
group.previewGroupPosition();
```